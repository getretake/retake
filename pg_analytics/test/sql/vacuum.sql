DROP TABLE IF EXISTS t;
DROP TABLE IF EXISTS s;
CREATE TABLE t (a int) USING deltalake;
CREATE TABLE s (a int);
INSERT INTO t VALUES (1), (2), (3);
INSERT INTO s VALUES (4), (5), (6);
VACUUM;
VACUUM FULL;
VACUUM t;
VACUUM FULL t;
DROP TABLE t, s;
VACUUM;
