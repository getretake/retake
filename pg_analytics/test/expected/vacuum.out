DROP TABLE IF EXISTS t;
DROP TABLE IF EXISTS s;
NOTICE:  table "s" does not exist, skipping
CREATE TABLE t (a int) USING parquet;
CREATE TABLE s (a int);
INSERT INTO t VALUES (1), (2), (3);
INSERT INTO s VALUES (4), (5), (6);
VACUUM;
VACUUM FULL;
VACUUM t;
VACUUM FULL t;
DROP TABLE t, s;
VACUUM;
